Src/WitsmlExplorer.Api/Models/ConnectionInformation.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WitsmlExplorer.Api.Models
+{
+    public class ConnectionInformation
+    {
+        [JsonPropertyName("url")]
+        public Uri Url { get; init; }
```

Comment:
Can you change this field to ServerUrl just to make it even more clear that this is connected to witsml server urls?